pubpub/pubpub
# feature: Spubheader Submission tab 'Title, Description & Abstract' sub-tab

Implements the functionality of the title, description, and abstract editing view as [laid out in the mock-ups](https://www.figma.com/file/ERHvqJqCVBZ7dH2ZQBGlp8/SPUB-vision?node-id=297%3A1386) for #1729

![Screenshot from 2022-02-22 11-58-43](https://user-images.githubusercontent.com/14115194/155180866-0c2b0298-cf7b-4bb6-8a72-4834a21264fb.png)


Todo: character remaining count + limits. 

_Test plan_
1. Visit a draft submission (or create one)
2. Navigate to the Submission tab
2. Navigate to the Title, Description & Abstract sub-tab
3. Edit the title, description, & abstract
4. Refresh the page, confirm that all your changes persist
